components/suggest/src/store.rs

```diff
@@ -337,16 +340,43 @@ where
                     .get_records_with_options(&options)?
                     .records;
 
-                self.ingest_records(writer, &records_chunk)?;
+                self.ingest_records(LAST_INGEST_META_UNPARSABLE, writer, &records_chunk)?;
             }
         }
 
+        let ingest_record_types = if let Some(rt) = &constraints.providers {
+            rt.iter().flat_map(|x| x.records_for_provider()).collect()
```

Comment:
Do we want to dedup the types in the result vector? `Icon` and `GlobalConfig` are shared by many providers, would it re-ingest those record types w/o deduping?

Reply:
Oh ya, there was a comment there originally about doing the dedupe. I'll see if I can add it easily.